### ISSUE TITLE
Incompatibility with Electron 21

### ISSUE THREAD

**Issue by zegenie** [ceifa/steamworks.js#51]
Thanks so much for finally taking on the task of building a modern steamworks api.

We're trying to use this library in an Electron 21-wrapped application, but have run into a compatibility issue with the new V8 memory cage as described here: https://www.electronjs.org/blog/v8-memory-cage

This causes Chromium/Electron to fatal crash whenever we're trying to for example retrieve the session ticket via the api:
`[167373:1105/112515.909269:ERROR:node_bindings.cc(146)] Fatal error in V8: v8_ArrayBuffer_NewBackingStore When the V8 Sandbox is enabled, ArrayBuffer backing stores must be allocated inside the sandbox address space. Please use an appropriate ArrayBuffer::Allocator to allocate these buffers`

As a result, it is impossible to use this as a node module in Electron 21 and onwards because of this. Trying to disable the sandbox either via launch parameters or `sandbox: false` has no effect on this setting. Apparently this issue needs to be fixed at the module level as described in the link above, where it mentions this:

> This change only affects native Node modules which allocate memory outside of the V8 heap (e.g. using malloc or new) and then wrap the external memory with an ArrayBuffer. This is a fairly rare use case, but some modules do use this technique, and such modules will need to be refactored in order to be compatible with Electron 20+

I don't know enough about the internals to know how to suggest this fix in a PR, or even where to suggest it, but the end result is still that it renders this module incompatible with Electron 20+ :disappointed:

**Comment by zegenie** [comment]
The issue arises when trying to call `ticket.getBytes()` on the value passed from the `auth.getSessionTicket()` promise:
```
const steam_appid = <our_steam_appid>;
const steamclient = steamworks.init(steam_appid);
steamclient.auth.getSessionTicket()
        .then(ticket => {
            console.info(ticket.getBytes());
        });
```

This code works in Electron 20, but from Electron 21 it causes a fatal V8 error.

**Comment by ceifa** [comment]
Related https://github.com/napi-rs/napi-rs/issues/1346

**Comment by James226** [comment]
I've tested this with the latest version of napi-rs now https://github.com/napi-rs/napi-rs/issues/1346 is resolved, and this seems to have resolved the issue. Any chance we can get a rebuild?

**Comment by oyed** [comment]
Would love a rebuild too, need to start upgrading Electron!

**Comment by ceifa** [comment]
Fixed on 0.0.27
